packages/tools/client-debugger/client-debugger-chrome-extension/src/test/ClientUtilities.ts

```diff
@@ -0,0 +1,127 @@
+/*!
```

Comment:
I would recommend waiting for #13604 to merge and pull those changes in. We will probably not need to add this file, and can instead use the Container mocks that PR adds.

Reply:
I see! Revert this change